transformer/images.js

```diff
@@ -299,10 +224,14 @@ async function resizeImage (jpegFilename) {
     let resizedFilename = `resized-${size.width ? `w${size.width}` : `h${size.height}`}${ext}`
     resizedFilename = path.join(path.dirname(jpegFilename), resizedFilename)
 
+    // TODO / rob note: I removed this check for existing files becuase if images are re-ordered then
+    // there will be collisions in the filenames. I think a potential solution (if the builds are too slow) is to have an
+    // ever-incrementing modification_index in the yml that we use to build the image directory name as well.
+
     // if resized file exists, don't overwrite
-    if (fs.existsSync(resizedFilename)) {
-      return Promise.resolve(resizedFilename)
-    }
+    // if (fs.existsSync(resizedFilename)) {
+    //   return Promise.resolve(resizedFilename)
+    // }
```

Comment:
Can you elaborate on this case? I agree that the check is irrelevant but I think it is because no files are ever being written back to git - so there would be nothing to overwrite. Sorry if I am misunderstanding - sleepy :)